tools/serverpod_cli/lib/src/analyzer/dart/endpoints_analyzer.dart

```diff
@@ -43,8 +43,7 @@ class EndpointsAnalyzer {
     var oldDefinitions = _endpointDefinitions;
     await analyze(collector: CodeGenerationCollector());
 
-    var shared = _endpointDefinitions.intersection(oldDefinitions);
-    if (shared.length != oldDefinitions.length) {
+    if (_endpointDefinitions.length != oldDefinitions.length) {
```

Comment:
This is a naive check but will work for now. If any endpoint file has been changed this will be detected in the "file" inspection.